src/frontend/src/utils/statusList/types.ts

```diff
@@ -3,10 +3,22 @@ import { SubmitButtonTextEnum } from '../../constant/constant';
 export type PageAccessControl = {
   [page: string]: {
     accessControl: boolean;
-    params?: any;
+    params?: any & PageParamProperties;
```

Comment:
Do we need 'any' ?

Reply:
We don't if we're going to explicitly define them from now on